portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,18 +27,28 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> comments;
+  private List comments = new ArrayList<>();
```

Comment:
Better to type this parameter, no?
https://stackoverflow.com/questions/3936556/when-you-make-an-arraylist-without-specifying-the-object-type-does-it-create-it